# Define the goal
uint32 distance # in cm, wie weit gefahren werden soll 
float32 speed	#in m pro sekunde
bool stop       #wenn true wird fahrt gestopped
---
# Define the result
uint32 distance_driven
geometry_msgs/Point position
bool canceled	#true, wenn abgebrochen, z.B. durch Wand
---
# Define a feedback message
float32 percent_complete